medical_patient_ethnicity/models/medical_patient_ethnicity.py

```diff
@@ -23,8 +23,8 @@
 from openerp import models, fields
 
 
-class MedicalEthnicity(models.Model):
-    _name = 'medical.ethnicity'
+class MedicalPatientEthnicity(models.Model):
+    _name = 'medical.patient.ethnicity'
 
     notes = fields.Char(size=256, string='Notes')
     code = fields.Char(size=256, string='Code')
```

Comment:
size attribute can be removed and string attribute too as it is the same value than the field name